code/modules/reagents/reagent_containers/food/snacks/eggs.dm

```diff
@@ -104,6 +104,9 @@
 	reagents_to_add = list(/singleton/reagent/nutriment/protein/egg = 6) // Big egg
 	hatchling = /mob/living/simple_animal/schlorrgo/baby
 
+/obj/item/reagent_containers/food/snacks/egg/schlorrgo/fertile/Initialize(mapload, fertile) // Snowflake subtype due to cargo code unable to insert values into an Initialize
+	return ..(mapload, TRUE)
```

Comment:
Not an acceptable way to do this, conver the param and use a var `fertile` instead of overriding /Initialize if you're gonna do subtypes

Reply:
Now have it so fertile is a var that is checked on init to fertilize. Setting var good for mapped in eggs, otherwise they can call the `fertilize` proc. Idea is in the future allow a way for eggs to be artificially fertilized (or perhaps sterilized)